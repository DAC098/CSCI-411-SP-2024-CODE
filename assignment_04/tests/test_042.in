4
1 2 2
aca tccactccccca
iinkllrriqpofhe oduqltpuxezrettckdzimrkczojdz
tcataaa acaaccccctccaccgccg
rqzurlxwze brzllcsumaxkvky
