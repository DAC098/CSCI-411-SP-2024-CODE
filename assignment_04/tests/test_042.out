___a______ca
tccactccccca
9
iinkl_____lr_________riqpofhe
oduqltpuxezrettckdzimrkczojdz
36
tc_a_____t__a____aa
acaaccccctccaccgccg
18
_rqz____urlx_wze
br_zllcsumaxkvky
18
